.github/README.md

```diff
@@ -0,0 +1,55 @@
+# [Oppia-Android](https://www.oppia.org)
+
+Oppia-Android is a mobile application version of [Oppia](https://www.oppia.org). It is an android application that enables anyone to learn easily interactive activities (called 'explorations'). These activities simulate a one-on-one conversation with a tutor, making it possible for students to learn by doing while getting feedback.
+
+Moreover, users can create their profile, practice a set of skills, play with explorations, review previously finished skills and also track the progress.
+
+Oppia-Android app is written in Kotlin and python, and is built on top of Google App Engine.
+
+  * [Oppia.org community site](https://www.oppia.org)
+  * [User Documentation](https://oppia.github.io/)
+  * [Contributors' wiki](https://github.com/oppia/oppia-android/wiki)
+  * [Developer mailing list](http://groups.google.com/group/oppia-android-dev)
+  * [File an issue](https://github.com/oppia/oppia-android/issues/new/)
+
+<p align="center">
+  <img width="361" alt="Screenshot 2019-12-18 at 4 37 57 PM" src="https://user-images.githubusercontent.com/9396084/71081269-ddf6f280-21b4-11ea-9a49-fcbd65b4ced7.png">
+</p>
+
+
+## Installation
+
+Please refer to the [developer wiki](https://github.com/oppia/oppia-android/wiki) for full installation instructions. This is just a summary for developers who would like to contribute:
+
+1. Create a new, empty folder called `opensource/` within your home folder. Navigate to it (`cd opensource`), then [fork and clone](https://github.com/oppia/oppia-android/wiki/Fork-and-Clone-Oppia) the Oppia repo. This will create a new folder named `opensource/oppia`.
```

Comment:
The link provided here does not exist. The folder name that is created is wrong.

Reply:
Updated to `https://github.com/oppia/oppia/wiki/Fork-and-Clone-Oppia`

Reply:
No, wrong link. That's not relevant to this repo.

Reply:
@seanlip Actually, we will need fork and clone in oppia-android too. So should we create a similar page in oppia-android too?

Reply:
Created `Fork and clone` page for `oppia-android`.